wafer/talks/admin.py

```diff
@@ -71,6 +80,37 @@ def has_delete_permission(self, request, obj=None):
         return False
 
 
+def mark_talks(value, fname):
+    def f(modeladmin, request, queryset):
+        for talk in queryset:
+            talk.status = value
+            talk.save()
```

Comment:
I think this can be `queryset.update(status=value)`

Reply:
I tried that at first, but with that I get a very strange error:

django.core.exceptions.FieldError

django.core.exceptions.FieldError: Cannot resolve keyword 'review_score_annotation' into field. Choices are: _abstract_rendered, abstract, authors, corresponding_author, corresponding_author_id, kv, notes, private_notes, reviews, scheduleitem, status, submission_time, talk_id, talk_type, talk_type_id, title, track, track_id, urls, video, video_reviewer

this is probably because the queryset contains annotations

Reply:
Right